manager/manager.go

```diff
@@ -308,6 +311,7 @@ func (m *Manager) Run(parent context.Context) error {
 	api.RegisterRaftMembershipServer(m.server, proxyRaftMembershipAPI)
 	api.RegisterControlServer(m.localserver, localProxyControlAPI)
 	api.RegisterControlServer(m.server, authenticatedControlAPI)
+	api.RegisterResourceAllocatorServer(m.server, authenticatedResourceAPI)
```

Comment:
Hmm.. May be I misunderstanding something. Doesn't the follower manager which forwards the requests do it automatically? Is anything need to be setup? I thought the only other thing needed is a localProxy for un-authenticated unix socket listener for direct communication with the leader.